paging-runtime-composeui/src/commonMain/kotlin/app/cash/paging/composeui/LazyPagingItems.kt

```diff
@@ -0,0 +1,318 @@
+package app.cash.paging.composeui
```

Comment:
It looks like this has been copy-pasted from the latest `paging-compose` release (i.e., `1.0.0-alpha17`) which depends on `paging-common:3.2.0-alpha03`. `paging-compose` needs to be compatible with Multiplatform Paging's version of `paging-common` (i.e., `3.1.1`).

Annoyingly, it doesn't look like any of the `paging-compose` releases depend on `paging-common:3.1.1`. `paging-compose:1.0.0-alpha14` depends on `paging-common:3.1.0-beta01` and `paging-compose:1.0.0-alpha15` depends on `paging-common:3.2.0-alpha01`, completely bypassing `paging-common:3.1.1`.

As a core goal of Multiplatform Paging is to be a drop-in replacement for AndroidX Paging on Android, we should have Multiplatform Paging support a version of `paging-common` that `paging-compose` wants. I'll add support for `paging-common:3.2.0-alpha03` so we can use the latest version of `paging-compose`.

Reply:
The branch [`main-3.2.0-alpha03`](https://github.com/cashapp/multiplatform-paging/tree/main-3.2.0-alpha03) is available! If you change the base branch from `main` to `main-3.2.0-alpha03`, that should be enough to solve this.